controller/requirements.txt

```diff
@@ -10,8 +10,8 @@ django-fsm==2.2.0
 django-guardian==1.2.5
 django-json-field==0.5.7
 djangorestframework==3.0.5
-docker-py==0.7.2
 gunicorn==19.3.0
+docker-py==1.0.0
```

Comment:
[docker-py 1.1.0](https://pypi.python.org/pypi/docker-py/1.1.0) is out now, just FYI.